db/schema.rb

```diff
@@ -64,6 +63,20 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "solutions", force: :cascade do |t|
+    t.string "title"
+    t.text "detail"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "solutons", force: :cascade do |t|
+    t.string "title"
+    t.text "detail"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
```

Comment:
ここも不要だと思うので削除した方がいいと思います。
```suggestion
  create_table "solutons", force: :cascade do |t|
    t.string "title"
    t.text "detail"
    t.datetime "created_at", precision: 6, null: false
    t.datetime "updated_at", precision: 6, null: false
  end
```

Reply:
こちらも不要だったので消しておきました！